packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -6,18 +6,20 @@ This package provides a collection of common values that are used in storybook b
 
 `LANGUAGE_VARIANTS` - A list of text samples in different languages, with the script and direction that should be used for that language.
 
-`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes two arguments, `slots` and `renderFn`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `renderFn`. This should usually be a React Component.
+`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```

Comment:
>Takes 4 arguments

Really it takes 1 arguement, an object with 4 possible keys 🙈

Reply:
Good spot @dr3